packages/web/src/components/pages/WebsiteResults.tsx

```diff
@@ -87,7 +114,26 @@ export function WebsiteResultsPage() {
         dispatch(resetScanDisplayOptions({ scanUrl: normalizedUrl }));
       }
     },
-    [dispatch, normalizedUrl]
+    [dispatch, normalizedUrl, selectedDisplayOptions]
+  );
+
+  const handleSortersChange = useCallback(
+    (newSorters: PackageSorter[]) => {
+      if (!normalizedUrl) {
+        return;
+      }
+
+      dispatch(
+        setScanDisplayOptions({
+          scanUrl: normalizedUrl,
+          options: {
+            ...selectedDisplayOptions,
+            packageSorters: newSorters,
+          },
+        })
+      );
+    },
+    [dispatch, normalizedUrl, selectedDisplayOptions]
```

Comment:
The two new handlers look very much alike. Instead of these, it would probably be better to write some additional reducers in the scanDisplayOptions slice, which will allow avoiding overwriting the whole options object here.